yarn.lock

```diff
@@ -0,0 +1,1208 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Wasn't sure if this should be gitignored or checked in.  🤷‍♂️

Reply:
I vote for ignore for a "gem" a la `Gemfile.lock` style. I think we will thrash it around otherwise.